code/datums/callback.dm

```diff
@@ -122,8 +121,8 @@
 	//if(datum_flags & DF_VAR_EDITED)
 	//	return WrapAdminProcCall(object, delegate, calling_arguments)
 	if (object == GLOBAL_PROC)
-		return call(delegate)(arglist(calling_arguments))
-	return call(object, delegate)(arglist(calling_arguments))
+		call(delegate)(arglist(calling_arguments))
+	call(object, delegate)(arglist(calling_arguments))
```

Comment:
wrong line to change, but honestly i don't get the return call here ?
@TiviPlus

Reply:
@BraveMole its to get returns from async calls but that should NOT be happening as its async